packages/bpk-component-loading-button/src/BpkLoadingButton.js

```diff
@@ -64,7 +68,23 @@ const getDefaultIcon = props => {
   return getEnabledIcon(large);
 };
 
-const BpkLoadingButton = props => {
+type LoadingProps = {
+  children: Node,
+  className: ?string,
+  disabled: boolean,
+  secondary: boolean,
+  destructive: boolean,
+  large: boolean,
+  link: boolean,
+  loading: boolean,
+  iconOnly: boolean,
+  icon: ?any,
```

Comment:
We try not to use `any`, especially in code we ship to users. How come it's not something more specific here? (Same question with `iconDisabled` and `iconLoading` props too)

Reply:
Ah good spot I just did this temp whilst doing work but forgot about it

Reply:
Done the correct changes